content/get-started/installation/streamlit-playground.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Use Streamlit Playground in your browser
+slug: /get-started/installation/streamlit-playground
+---
+
+# Use Streamlit Playground in your browser
+
+The fastest way to try out Streamlit is to try out our Playground! Streamlit Playground runs in your browser. Just visit the Playground, and a _limited_ version of Streamlit loads as browser scripts.
+
+Enjoy the following conveniences:
+
+- Start playing with code right away, completely in your browser.
+- No configuration.
+- No command line.
+- No application installations in your OS.
+
+<Important>    
+    Although the Playground has everything you need to get started, it doesn't contain the full version of Streamlit. To access the full awesomeness of Streamlit, see [Install Streamlit using command line](/get-started/installation/command-line) or [Install Streamlit using Anaconda Distribution](/get-started/installation/anaconda-distribution).
+</Important>
+
+## Prerequisites
+
+Because the Playground runs Streamlit _locally_ in your browser, you should visit the Playground from a personal computer, not a mobile device.
+
+## Go to the Playground
+
+1. Go to [streamlit.io/playground](https://streamlit.io/playground).
+
+1. Wait for [`stlite`](https://github.com/whitphx/stlite) &mdash; a browser-based version of Streamlit &mdash; to load in your browser.
```

Comment:
I don't think you should mention stlite here. It's just confusing to the user.

Instead just say something like: `Wait for the playground to load. Behind the scenes this downloads a version of Python and Streamlit that your browser can run, so this may take a bit of time.`